h5p-bildetema/src/components/TopicRouteController/TopicRouteController.module.scss

```
@@ -0,0 +1,25 @@
+@use "../../../../common/abstracts" as *;
+
+.body {
+  padding: 2rem 0;
```

Only add padding to bottom and not top
```suggestion
  padding: 0 0 2rem;
```